### PR TITLE
upgrade rubocop version

### PR DESCRIPTION
**Pull request by namusyaka**
Fixes potential security vulnerability.
ref https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-8418

### CODE REVIEW

**Comment by dblock** [comment]
You should upgrade to the latest rubocop, fix appraisals and what not, too. Then `rubocop -a ; rubocop --auto-gen-config`.

**Comment by namusyaka** [comment]
Fixed.

**Comment by dblock** [comment]
Thanks. The encoding header was removed everywhere, which is right, but a newline was added instead. Care to fix that? Sorry to be a pest :)

File: lib/grape/exceptions/incompatible_option_values.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# encoding: utf-8
+
```
Comment (dblock) [inline lib/grape/exceptions/incompatible_option_values.rb]: This newline was added.

**Comment by namusyaka** [comment]
Oh.. sorry. Thanks for pointing this out.

**Pull request merged**